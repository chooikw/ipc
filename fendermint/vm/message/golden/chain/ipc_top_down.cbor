a163497063a16b546f70446f776e45786563a26870726f706f73616ca1625631a4666865696768741ae60ba20e6a626c6f636b5f6861736887186c18d9187b187318f7186f18326e63726f73735f6d65737361676573807176616c696461746f725f6368616e676573806463657274a2777369676e65645f76616c696461746f725f6269746d6170a4656f72646572736269747665633a3a6f726465723a3a4c7362306468656164a2657769647468184065696e646578006462697473006464617461806e6167675f7369676e617475726573a165456364736180