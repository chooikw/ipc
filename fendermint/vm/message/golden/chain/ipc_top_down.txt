Ipc(TopDownExec(ParentFinality { height: 4089175392, block_hash: [152] }))